### PR TITLE
Flavour static page 

### PR DESCRIPTION
**Pull request by MousaRaza7**
image
I added the static flavour page in a seperate folder named flavours-static-draft for review. I have also added an transparent logo and the whiskeys images for the static flavour pages. I also added an vanillin chemical image which is a duplicate so don't worry about it.

### CODE REVIEW

File: next-app/public/images/chemicals/vanillin.png
Comment (mattbeard03) [inline next-app/public/images/chemicals/vanillin.png:1]: Not required.

File: next-app/public/images/logo/Logo.png
Comment (mattbeard03) [inline next-app/public/images/logo/Logo.png:1]: Not Required.

**Review by mattbeard03 — CHANGES_REQUESTED** [review]

File: next-app/public/images/chemicals/vanillin.png
Comment (MousaRaza7) [reply next-app/public/images/chemicals/vanillin.png:1]: Removed

File: next-app/public/images/logo/Logo.png
Comment (MousaRaza7) [reply next-app/public/images/logo/Logo.png:1]: Removed

**Comment by ngu892** [comment]
is it for individual flavours page?

**Comment by MousaRaza7** [comment]
> is it for individual flavours page?

yes

**Pull request closed without merging**